public-ui/src/app/data-browser/views/genomic-view/components/variant-table.component.tsx

```diff
@@ -1,73 +1,117 @@
-        }).catch(e => {
-            console.log(e, 'error');
-        });
-    }
-
     render() {
-        const {loading, numPages} = this.state;
-        return <React.Fragment>
-            <div style={styles.border}>
-                <p>I am Variant Table</p>
-                <VariantRowComponent />
-                                {!loading &&
-                                        <ReactPaginate
-                                            previousLabel={'Previous'}
-                                            nextLabel={'Next'}
-                                            breakLabel={'...'}
-                                            breakClassName={'break-me'}
-                                            activeClassName={'active'}
-                                            pageCount={numPages}
-                                            marginPagesDisplayed={2}
-                                            pageRangeDisplayed={5}
-                                            onPageChange={this.handlePageClick}
-                                            containerClassName={'pagination'}
-                                        />}
+        const { numPages } = this.state;
+        const { searchResults, variantListSize } = this.props;
+        return <React.Fragment> {searchResults &&
+            <div style={styles.tableContainer}>
+                <div style={styles.headerLayout}>
+                    <div style={{ ...styles.headingItem, ...styles.first }}><span style={styles.headingLabel}>Variant ID</span></div>
+                    <div style={styles.headingItem}><span style={styles.headingLabel}>Gene</span></div>
+                    <div style={styles.headingItem}><span style={styles.headingLabel}>Consequence</span></div>
+                    <div style={styles.headingItem}><span style={styles.headingLabel}>Protein Change</span></div>
+                    <div style={styles.headingItem}><span style={styles.headingLabel}>Clinical Significance</span></div>
+                    <div style={styles.headingItem}><span style={styles.headingLabel}>Allele Count</span></div>
+                    <div style={styles.headingItem}><span style={styles.headingLabel}>Allele Number</span></div>
+                    <div style={{ ...styles.headingItem, ...styles.last }}><span style={styles.headingLabel}>Allele Frequency</span></div>
+                </div>
+                {searchResults && searchResults.map((varData, index) => {
+                    return <VariantRowComponent key={index} varData={varData} />;
+                })}
+
+                {variantListSize &&
+                    <ReactPaginate
+                        previousLabel={'Previous'}
+                        nextLabel={'Next'}
+                        breakLabel={'...'}
+                        breakClassName={'break-me'}
+                        activeClassName={'active'}
+                        pageCount={numPages}
```

Comment:
I think we need to use `variantListSize` to get the page count. Currently it's always 0.

Reply:
I want to keep it I'm pretty sure we will need it later.